make_ghpages/make_pages.py

```diff
@@ -134,6 +148,33 @@ def get_html_provider_fname(provider_id):
     return "{}.html".format(simple_string)
 
 
+def validate_childdb(url: str) -> dict:
+    """ Run the optimade-python-tools validator on the child database.
+
+    Parameters:
+        url: the URL of the child database.
+
+    Returns:
+        dictionary representation of the validation results.
+
+    """
+    import dataclasses
+    from traceback import print_exc
+    validator = ImplementationValidator(
+        base_url=url,
+        run_optional_tests=False,
+        verbosity=-1,
+        fail_fast=False
+    )
+
+    try:
+        validator.validate_implementation()
+    except Exception:
+        print_exc()
```

Comment:
```suggestion
    validator.validate_implementation()
```

Reply:
The validator can currently raise `RuntimeError` if it fails in the initial stages (e.g. the provided URL 404s entirely) so it does need to be caught; we might consider changing this in the validator (see next comment about `verbosity=-1` too)

Reply:
This is a bit serious, I think, since we shouldn't have to do the trick you're doing here.